protos/timer.proto

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+syntax = "proto3";
+
+message TimerData {
+  uint64 start = 1;
+  uint64 end = 2;
+  int32 process_id = 3;
+  int32 thread_id = 4;
+  uint32 depth = 5;
+
+  enum Type {
+    kNone = 0;
+    kCoreActivity = 1;
+    kIntrospection = 2;
+    kGpuActivity = 3;
+  }
+  Type type = 6;
+
+  int32 processor = 7;
+  uint64 callstack_hash = 8;
+  uint64 function_address = 9;
+  uint64 user_data_key = 10;
+  uint64 timeline_hash = 11;
+}
```

Comment:
@dpallotti I see your comment about completely changing the TimerData in the previous PR, but I would prefer to postpone it for now, because it requires a lot of changes